modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -168,7 +168,7 @@
     nil))
 
 (defn- database-type->temporal-type [database-type]
-  (condp = (some-> database-type str/upper-case)
+  (condp = (some-> database-type u/upper-case-en)
```

Comment:
great catch. I'm surprised no one ever ran into errors related to this yet

we really need to write some sort of Kondo linter to ban `str/upper-case` or something... or maybe we should just hack it and replace it with `u/upper-case-en`